CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 ### Breaking changes
 * The SchemaMode::Additive has been replaced by two different modes: AdditiveDiscovered and AdditiveExplicit. The former should be used when the schema has been automatically discovered, and the latter should be used when the user has explicitly included the types in the schema. Different schema checks are enforced for each scenario. ([#4306](https://github.com/realm/realm-core/pull/4306))
 * Revert change in `app::Response` ([4263](https://github.com/realm/realm-core/pull/4263))
+* File format version bumped to 21.
```

Comment:
I think the changelog should state the differences between the new and previous file format in a bit more detail.

Reply:
Like now?